src/include/executor/executor_context.h

```diff
@@ -30,7 +30,8 @@ namespace executor {
 class ExecutorContext {
  public:
   explicit ExecutorContext(concurrency::TransactionContext *transaction,
-                           codegen::QueryParameters parameters = {});
+                           codegen::QueryParameters parameters = {},
+                           std::string default_database_name = "");
```

Comment:
Is this appropriate to add default_database_name here? Shouldn't plans contain all the necessary information?